gm4_holographic_tags/data/holographic_tags/functions/main.mcfunction

```diff
@@ -0,0 +1,3 @@
+execute as @e[type=area_effect_cloud,tag=gm4_hologram] at @s positioned ^ ^ ^-0.3 positioned ~ ~.3 ~ if entity @e[type=item_frame,nbt=!{Item:{}},distance=..0.1,limit=1] run function holographic_tags:destroy_hologram
+execute as @e[type=item_frame,tag=!gm4_hologram] if data entity @s {Item:{id:"minecraft:name_tag",tag:{display:{}}}} at @s positioned ^ ^ ^0.3 positioned ~ ~-.3 ~ unless entity @e[type=area_effect_cloud,tag=gm4_hologram,distance=..0.1] run function holographic_tags:create_hologram
+tag @e[type=item_frame,nbt=!{Item:{id:"minecraft:name_tag"}}] remove gm4_hologram
```

Comment:
Check for the `gm4_hologram` tag before the nbt to exclude most item frames from the nbt check
```suggestion
tag @e[type=item_frame,tag=gm4_hologram,nbt=!{Item:{id:"minecraft:name_tag"}}] remove gm4_hologram
```